server/backends/blobstore/blobstore.go

```diff
@@ -39,10 +39,12 @@ import (
 
 var (
 	// Disk flags
-	rootDirectory     = flag.String("storage.disk.root_directory", "/tmp/buildbuddy", "The root directory to store all blobs in, if using disk based storage.")
-	partitions        []disk.Partition
-	partitionMappings []disk.PartitionMapping
-	useV2Layout       = flag.Bool("storage.disk.use_v2_layout", false, "If enabled, files will be stored using the v2 layout. See disk_cache.MigrateToV2Layout for a description.")
+	rootDirectory = flag.String("storage.disk.root_directory", "/tmp/buildbuddy", "The root directory to store all blobs in, if using disk based storage.")
+	useV2Layout   = flag.Bool("storage.disk.use_v2_layout", false, "If enabled, files will be stored using the v2 layout. See disk_cache.MigrateToV2Layout for a description.")
```

Comment:
feel free to delete these 3 flags (use_v2_layout, partitions, partition_mappings), they are only applicable to cache storage

Reply:
Done!